packages/next-ui/Theme/MuiFab.ts

```diff
@@ -28,7 +28,7 @@ const defaultSizes: FabSizes = {
   small: '40px',
   medium: '48px',
   large: '56px',
-  responsive: responsiveVal(40, 56),
+  responsive: responsiveVal(40, 54),
```

Comment:
The responsive one matches the small-to-large variants. And I believe the large variant currently is just the default MUI size, keep it at that size?